csharp/ql/lib/semmle/code/csharp/frameworks/system/io/StreamReader.qll

```diff
@@ -0,0 +1,31 @@
+/** Provides definitions related to `System.IO.StreamReader`. */
+
+import csharp
+private import semmle.code.csharp.frameworks.system.IO
+private import semmle.code.csharp.dataflow.ExternalFlow
+
+/** The `System.IO.StreamReader` class. */
+class SystemIOStreamReader extends SystemIOClass {
+  SystemIOStreamReader() { this.hasName("StreamReader") }
+}
+
+/** Data flow for `System.IO.StreamReader`. */
+private class SystemIOStreamReaderFlowModelCsv extends SummaryModelCsv {
+  override predicate row(string row) {
+    row =
+      [
+        "System.IO;StreamReader;false;StreamReader;(System.IO.Stream);;Argument[0];Argument[Qualifier];taint",
```

Comment:
Please note that the content of this [PR](https://github.com/github/codeql/pull/9025) changes the CSV format. All the entered rows here needs to be postfixed with ";manual"

Reply:
Thanks, for the heads up. I'll rebase. I expect to get back to this in a week.